certified-connectors/Gratavid/apiDefinition.swagger.json

```diff
@@ -0,0 +1,542 @@
+            "in": "query",
+            "required": false,
+            "type": "integer"
+          },
+          {
+            "name": "webookHookEvent",
+            "in": "query",
+            "required": false,
+            "type": "string"
+          }
+        ]
+      },
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {}
+          }
+        },
+        "summary": "When a new event (watch, reply, etc.) takes place in Gratavid",
+        "description": "Triggers when a new event such as Gratavid watch, reply, task sent, unsubscribe, or text-opt out.",
+        "operationId": "newEvent",
+        "x-ms-trigger": "single",
+        "parameters": [
+          {
+            "name": "source",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "microsoftPowerAutomate",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "event",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "default": "webhookSubscribe",
+            "x-ms-visibility": "internal"
+          },
+          {
+            "name": "webookHookEvent",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "enum": [
+              "taskSent",
+              "reply",
+              "noteWatched",
+              "unsubscribe",
+              "stopText"
+            ],
+            "x-ms-enum-values": [
+              {
+                "displayName": "Gratavid task sent to a contact",
+                "value": "taskSent"
+              },
+              {
+                "displayName": "Contact replies to a Gratavid",
+                "value": "reply"
+              },
+              {
+                "displayName": "Contact watches a Gratavid all the way through",
+                "value": "noteWatched"
+              },
+              {
+                "displayName": "Contact unsubscribes from email after receiving Gratavid email",
+                "value": "unsubscribe"
+              },
+              {
+                "displayName": "Contact opt-out of text (replies 'STOP') after receiving Gratavid",
+                "value": "stopText"
+              }
+            ],
+            "x-ms-visibility": "important",
+            "description": "Select the event you want to trigger on",
+            "x-ms-summary": "Gratavid Event"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "webhookURL": {
+                  "type": "string",
+                  "description": "webhookURL",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
```

Comment:
@Wtrapp Can you please let us know why the title value is given as an empty string ? If it is not required, it can be removed.

Reply:
Good catch. Removed in the latest commit.